File: tests/fixtures/corpus/c/playground.c
#include <stdbool.h>
#include <stdio.h>

void greet(const char *name, bool excited) {
    printf("Hello, %s%s\n", name, excited ? "!" : "");
}

int main(void) {
    greet("World", true);
    return 0;
}

File: tests/fixtures/corpus/python/playground.py
def greet(name: str) -> None:
    print(f"Hello, {name}")

def add(x: int, y: int) -> int:
    return x + y

class Counter:
    def __init__(self) -> None:
        self.count: int = 0

    def increment(self) -> None:
        self.count += 1
